src/silvimetric/resources/entry.py

```diff
@@ -26,9 +24,17 @@ def schema(self) -> Attr:
     def to_json(self) -> object:
         raise NotImplementedError
 
+    def toJSON(self) -> object:
```

Comment:
having both `to_json` and `toJSON` seems confusing, especially when one raises `NotImplementedError` and the other does stuff. Pick the one everyone else uses to do this